src/changelog/mergeChangelogs.ts

```diff
@@ -26,8 +26,11 @@ export function mergeChangelogs(
 
   changelogs.forEach(changelog => {
     (Object.keys(changelog.comments) as ChangeType[]).forEach(changeType => {
+
+
       if (changelog.comments[changeType]) {
-        result.comments[changeType] = (result.comments[changeType] || []).concat(changelog.comments[changeType]!);
```

Comment:
Unless I'm reading this wrong, but there's no meaningful changes to this file anymore, maybe you can just revert this file change.